### DISCUSSION TITLE
开源生态学初探——从生命游戏开始

### DISCUSSION THREAD

**Discussion by giscus** [zhuangbiaowei/zhuangbiaowei.github.io#19] [Announcements]
# 开源生态学初探——从生命游戏开始

缘起

https://zhuangbiaowei.github.io/2022/05/21/a-first-look-at-open-source-ecology-starting-with-the-game-of-life.html

**Comment by ZoomQuiet** [comment]
生命游戏, 好像已经变成了一个核心母题, 从中已经生成了很多论文...
最打动人心的, 就是其复杂过程和结果, 只基于非常小的规则,
这就等于可以对复杂社会期望能有几个底层规则来完成足够的再现...
可惜, 现实并不是这样的...
